### PR TITLE
Rework

### PR DESCRIPTION
**Pull request by kol-oss**

### CODE REVIEW

File: app/tempCodeRunnerFile.js [hunk 1]
```diff
@@ -0,0 +1 @@
+queue
```
Comment (kovaliovev) [inline app/tempCodeRunnerFile.js:1]: trash temp file, delete and add to .gitignore

**Review by kovaliovev — CHANGES_REQUESTED** [review]
All in all, looks good 🎉

File: app/1-shortest.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      minDistance = distance;
+      minNode = node;
+    }
+  }
+  return minNode;
+}
+
+function buildFullPath(target, distances) {
+  const path = [target];
+  let currentPrev = target;
```
[the remark this answers is not in the document]
Comment (kol-oss) [reply app/1-shortest.js]: Variable name was changed from "currentPrev" to "current"

File: app/tempCodeRunnerFile.js (hunk 1 above)
Comment (kol-oss) [reply app/tempCodeRunnerFile.js]: Trash file was deleted and added to gitignore file

**Review by kovaliovev — APPROVED** [review]

**Pull request merged**